src/utils/streams.js

```diff
@@ -0,0 +1,158 @@
+import program from "commander"
+import readline from "readline"
+import reverseString from "reverse-string"
+import fs from "fs"
+import csv from "csvtojson"
+import Deque from "collections/deque";
```

Comment:
Try to avoid usage of `collection` library, use native `Array, Set, Map` instead. If you're not familiar with them already, please check this
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Set
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Map